payments/views.py

```diff
@@ -103,6 +105,9 @@ def pay(request, order):
             order=order.pk, err=payment.error))
         messages.error(request, _('Something went wrong, try again later.'))
 
+        order.processing_fee -= paypal_fee
```

Comment:
Hmm.. this doesn't make sense to me.

Why are you adding the `paypal_fee` to `processing_fee` and then subtracting it?